student/src/main/java/tds/student/sql/repository/remote/impl/RemoteExamPageRepository.java

```diff
@@ -31,6 +40,41 @@ public RemoteExamPageRepository(@Qualifier("integrationRestTemplate") final Rest
     }
 
     @Override
+    public ExamPage findPageWithItems(final OpportunityInstance opportunityInstance, final int position) throws ReturnStatusException {
+        UriComponents uriComponents = UriComponentsBuilder.fromUriString("{examUrl}/{examId}/page{position}")
+            .queryParam("sessionId", opportunityInstance.getSessionKey())
+            .queryParam("browserId", opportunityInstance.getExamBrowserKey())
+            .buildAndExpand(examUrl, opportunityInstance.getExamId(), position);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<?> requestHttpEntity = new HttpEntity<>(headers);
+        ResponseEntity<Response<ExamPage>> responseEntity;
+
+        ExamPage examPage = null;
+
+        try {
+            responseEntity = restTemplate.exchange(uriComponents.encode().toUri(),
```

Comment:
Had to use `exchange` here instead of `getForEntity` or `getForObject` because the exam service endpoint returns a `Response<ExamPage>`, which `getForEntity`/`getForObject` have trouble dealing with (due to type erasure presumably, which the `ParameterizedTypeReferecne` resolves).  I thought about changing the exam service endpoint to just return an `ExamPage`, but didn't want to break any other callers or spend time to hunt down all locations that would be affected by the change.